extensions/wikia/Wall/Wall.setup.php

```diff
@@ -224,15 +224,21 @@
 $wgGroupPermissions['vstf']['wallarchive'] = true;
 $wgGroupPermissions['helper']['wallarchive'] = true;
 $wgGroupPermissions['sysop']['wallarchive'] = true;
+$wgGroupPermissions['moderator']['wallarchive'] = true;
 
 $wgGroupPermissions['*']['wallremove'] = false;
-$wgGroupPermissions['user']['wallremove'] = true;
+$wgGroupPermissions['user']['wallremove'] = false;
```

Comment:
Gotcha. I went ahead and added a pull request for that config change and added it to the ticket. We'll make sure to merge it at the same time as this one. Thanks for updating those lines.